generators/client-2/templates/src/main/webapp/app/shared/_shared.ng2module.ts

```diff
@@ -24,12 +34,19 @@ import {
 @NgModule({
     imports: [
         <%=angular2AppName%>SharedLibsModule,
-        <%=angular2AppName%>SharedCommonModule,
+        <%=angular2AppName%>SharedCommonModule<%_ if (enableTranslation){ _%>,
+        TranslateModule.forRoot({
+          provide: TranslateLoader,
+          useFactory: (http: Http) => new TranslateStaticLoader(http, '/i18n', '.json'),
```

Comment:
as per docs isnt this the default `TranslateStaticLoader` then do we have to overwrite it?